.evergreen/build_all.sh

```diff
@@ -36,7 +36,17 @@ command -v $CMAKE || (echo "CMake could not be found...aborting!"; exit 1)
 cd libmongocrypt
 mkdir cmake-build
 cd cmake-build
-$CMAKE -DCMAKE_BUILD_TYPE=RelWithDebInfo $ADDITIONAL_CMAKE_FLAGS "${LIBMONGOCRYPT_EXTRA_CMAKE_FLAGS}" -DCMAKE_C_FLAGS="${LIBMONGOCRYPT_EXTRA_CFLAGS}" -DCMAKE_PREFIX_PATH="${BSON_INSTALL_PREFIX}" "-DCMAKE_INSTALL_PREFIX=${MONGOCRYPT_INSTALL_PREFIX}" -DCMAKE_EXPORT_COMPILE_COMMANDS=ON ../
+
+if [ "$OS" = "Windows_NT" ]; then
+    # W4996 - POSIX name for this item is deprecated
+    # TODO: add support for clang-cl which is detected as MSVC
```

Comment:
clang-cl will work as-is The only caveat is that the warnings enable/disable is pretty hacky in clang-cl. A lot of the warnings switches in cl.exe are no-ops in clang-cl.

Reply:
I am just preserving a comment that was [already present](https://github.com/mongodb/libmongocrypt/blob/ddd2f105fe34ce7682fbec76c4a443f26f4ed076/CMakeLists.txt#L105). AFAIK these changes do nothing to address it. If it is no longer applicable or relevant, I can remove it as part of these changes.